Documentation/concepts/terminology.rst

```diff
@@ -202,36 +202,46 @@ order to allow communication to network endpoints which are not managed by
-| Identity            | Description                                       |
-+=====================+===================================================+
-| reserved:unknown    | The identity could not be derived.                |
-+---------------------+---------------------------------------------------+
-| reserved:host       | The collection of all cluster hosts. Any traffic  |
-|                     | that originates from or is designated to one of   |
-|                     | the IPs of any host in the cluster is assigned the|
-|                     | reserved:host identity.                           |
-+---------------------+---------------------------------------------------+
-| reserved:world      | Any network endpoint outside of the cluster       |
-+---------------------+---------------------------------------------------+
-| reserved:health     | This is health checking traffic generated by      |
-|                     | Cilium agents.                                    |
-+---------------------+---------------------------------------------------+
-| reserved:init       | An endpoint for which the identity has not yet    |
-|                     | been resolved is assigned the init identity.      |
-|                     | This represents the phase of an endpoint in which |
-|                     | some of the metadata required to derive the       |
-|                     | security identity is still missing. This is       |
-|                     | typically the case in the bootstrapping phase.    |
-|                     |                                                   |
-|                     | The init identity is only allocated if the labels |
-|                     | of the endpoint are not known at creation time.   |
-|                     | This can be the case for the Docker plugin.       |
-+---------------------+---------------------------------------------------+
-| reserved:unmanaged  | An endpoint that is not managed by Cilium, e.g.   |
-|                     | a Kubernetes pod that was launched before Cilium  |
-|                     | was installed.                                    |
-+---------------------+---------------------------------------------------+
++-------------------------+---------------------------------------------------+
+| Identity                | Description                                       |
++=========================+===================================================+
+| reserved:unknown        | The identity could not be derived.                |
++-------------------------+---------------------------------------------------+
+| reserved:host           | The local host. Any traffic that originates from  |
+|                         | or is designated to one of the local host IPs.    |
++-------------------------+---------------------------------------------------+
+| reserved:non-local-node | The collection of all non-local cluster hosts.    |
```

Comment:
hmmm this looks really similar to the `reserved:cluster` identity he had

Reply:
Yes. It's pretty similar to what we had before we moved the cluster to be an entity to include worker nodes, host, managed pods and unmanaged pods.